community/lua-bitop/APKBUILD

```diff
@@ -0,0 +1,51 @@
+# Contributor:
+# Maintainer:
+pkgname="lua-bitop"
+pkgver="1.0.2"
+pkgrel=0
+pkgdesc="Bitwise operations on numbers"
+url="https://bitop.luajit.org/"
+arch="all"
+license="MIT"
+makedepends=""
```

Comment:
empty variables can be removed for brevity

Reply:
Will do